freqtrade/constants.py

```diff
@@ -371,7 +371,9 @@
             'type': 'string',
             'enum': AVAILABLE_DATAHANDLERS,
             'default': 'jsongz'
-        }
+        },
+        'position_adjustment_enable': {'type': 'boolean', 'default': False},
```

Comment:
Both of these parameters shouldn't be in the config schema!

If they are in the schema and have defaults then they overwrite strategy, meaning people without `"position_adjustment_enable": true` in their config file will have it turned off regardless if the strategy says it should be True.

Reply:
already fixed in fb3c67d86b64c6d603de87a8b156e78398a661b7 :)

Reply:
good spot - fixed in e7409e74c2c4798c78559d47eb69842202fca288

Reply:
But it's not behaving that way? I'm using it live that way, and it work just fine (I don't enable the position adjustment on config.json, only on strategy file). The default don't overwrite my strategy

Reply:
Ah never mind. Just read perk's reply. It caused problem on backtest.

Reply:
it'll cause the same problem in live as well iirc - as the setting will come from the configuration already...